evaluate.py

```diff
@@ -179,7 +179,7 @@ def main(args: argparse.Namespace) -> None:
     elif issubclass(TASK, SemanticSegmentationTask):
```

Comment:
Same problem with chmod here

Reply:
I think that is from resolving the merge conflict inside github and committing from there.